Formula/m/mkvtoolnix.rb

```diff
@@ -27,29 +28,40 @@ class Mkvtoolnix < Formula
   end
 
   depends_on "docbook-xsl" => :build
+  depends_on "nlohmann-json" => :build
   depends_on "pkgconf" => :build
+  depends_on "utf8cpp" => :build
   depends_on "boost"
   depends_on "flac"
   depends_on "fmt"
-  depends_on "gettext"
   depends_on "gmp"
   depends_on "libebml"
   depends_on "libmatroska"
   depends_on "libogg"
   depends_on "libvorbis"
   # https://mkvtoolnix.download/downloads.html#macosx
   depends_on macos: :catalina # C++17
-  depends_on "nlohmann-json"
   depends_on "pugixml"
   depends_on "qt"
-  depends_on "utf8cpp"
 
   uses_from_macos "libxslt" => :build
   uses_from_macos "ruby" => :build
   uses_from_macos "zlib"
 
+  on_macos do
+    depends_on "gettext"
+  end
+
+  on_linux do
+    depends_on "gettext" => :build
+  end
```

Comment:
Seems like you could just make this unconditional given the `on_macos` block above.

Reply:
Yeah. Can update if another run is needed.